drivers/storage/portworx/manifest/manifest.go

```diff
@@ -33,7 +33,9 @@ const (
 	defaultPrometheusOperatorImage        = "quay.io/coreos/prometheus-operator:v0.34.0"
 	defaultPrometheusConfigMapReloadImage = "quay.io/coreos/configmap-reload:v0.0.1"
 	defaultPrometheusConfigReloaderImage  = "quay.io/coreos/prometheus-config-reloader:v0.34.0"
-	defaultManifestRefreshInterval        = 3 * time.Hour
+	// TODO change this once phone home team is done uploading image: https://jira.purestorage.com/browse/CLOUD-57162
```

Comment:
nit: add an OPERATOR bug as well so that we won't miss this

Reply:
added